superset-frontend/src/datasource/ChangeDatasourceModal.tsx

```diff
@@ -33,7 +33,7 @@ import withToasts from '../messageToasts/enhancers/withToasts';
 
 interface ChangeDatasourceModalProps {
   addDangerToast: (msg: string) => void;
-  onChange: (id: number) => void;
+  onChange: (uid: string) => void;
```

Comment:
Curious, what does `uid` mean? Is this supposed to be UUID?

Reply:
unique id i guess, and is always the `id__table`